codespell_lib/data/dictionary.txt

```diff
@@ -4840,7 +4841,7 @@ choser->chooser
 chosing->choosing
 chossen->chosen
 chould->should, could,
-chouse->chose
+chouse->choose, chose,
```

Comment:
Would `choux` also be a suitable suggestion?

Reply:
It's a bit niche, but added.